artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/jdbc/JdbcLeaseLock.java

```diff
@@ -114,14 +115,14 @@ private String readableLockStatus() {
          final boolean autoCommit = connection.getAutoCommit();
          connection.setAutoCommit(false);
          try (PreparedStatement preparedStatement = connection.prepareStatement(this.isLocked)) {
+            final long currentTimestamp = dbCurrentTimeMillis(connection);
```

Comment:
I guess in dbCurrentTimeMillis() we should select the current UTC timestamp query based on the database type (e.g. `current-timestamp.mssql`) and fall back to `current-timestamp` with an appropriate warning if the database is unknown.

Reply:
Yep, this is already how internally work the SQLProvider class donated by Jeff Mesnil from WildFly

Reply:
ok